pkgs/development/python-modules/pdfx/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, pdfminer, chardet, pytest }:
+
+buildPythonPackage rec {
+  pname = "pdfx";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
```

Comment:
I have opened https://github.com/metachris/pdfx/pull/29 so that `fetchPypi` can be used.